### PR TITLE
Syntax issues (paragraphs 122, 130, 147, 155) Documentation: @sgugger

### PR DESCRIPTION
**Pull request by kant**
# What does this PR do?
* Syntax issues (paragraphs 122, 130, 147, 155): `preentramiento` > `preentrenamiento`
* semantic issue (paragraph 220 & 232 & 252)

Fixes # (issue)
Resolves no previous issue

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@sgugger 
@omarespejel
@ignacioct

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/es/create_a_model.mdx [hunk 1]
```diff
@@ -144,15 +144,15 @@ Carga los atributos de tu configuración personalizada en el modelo de la siguie
 >>> tf_model = TFDistilBertModel(my_config)
 ```
 
-Esto crea un modelo con valores aleatorios, en lugar de crearlo con los pesos del preentramiento, por lo que no serás capaz de usar este modelo para nada útil hasta que no lo entrenes. El entrenamiento es un proceso costoso, tanto en cuestión de recursos como de tiempo, por lo que generalmente es mejor usar un modelo preentrenado para obtener mejores resultados más rápido, consumiendo solo una fracción de los recursos que un entrenamiento completo hubiera requerido. 
+Esto crea un modelo con valores aleatorios, en lugar de crearlo con los pesos del preentrenamiento, por lo que no serás capaz de usar este modelo para nada útil hasta que no lo entrenes. El entrenamiento es un proceso costoso, tanto en cuestión de recursos como de tiempo, por lo que generalmente es mejor usar un modelo preentrenado para obtener mejores resultados más rápidos, consumiendo solo una fracción de los recursos que un entrenamiento completo hubiera requerido.
```
Comment (ignacioct) [inline docs/source/es/create_a_model.mdx:147]:
```suggestion
Esto crea un modelo con valores aleatorios, en lugar de crearlo con los pesos del preentrenamiento, por lo que no serás capaz de usar este modelo para nada útil hasta que no lo entrenes. El entrenamiento es un proceso costoso, tanto en cuestión de recursos como de tiempo, por lo que generalmente es mejor usar un modelo preentrenado para obtener mejores resultados más rápido, consumiendo solo una fracción de los recursos que un entrenamiento completo hubiera requerido. 
```

Rápido se usa en singular porque es un complemento circunstancial, relacionado con el verbo, que esta en singular; no es un adjetivo de ¨resultados¨.

**Review by ignacioct — CHANGES_REQUESTED** [review]
Gracias por la corrección. Solo hay un detalle con el que no estoy de acuerdo, pero el resto esta perfecto :)

**Comment by sgugger** [comment]
Same as on your other PR, it seems there is an issue with your CircleCI permissions, the tests won't run.
Could you try refreshing your permissions as shown [here](https://support.circleci.com/hc/en-us/articles/360048210711-How-to-Refresh-User-Permissions-)?

File: docs/source/es/create_a_model.mdx (hunk 1 above)
Comment (kant) [reply docs/source/es/create_a_model.mdx:147]: You are right, @ignacioct

**Comment by kant** [comment]
Permissions refreshed @sgugger

**Comment by sgugger** [comment]
Hi @kant It didn't work but I should have fixed the issue on our side. Could you make sure to accept the suggestion above?

**Comment by kant** [comment]
I accept the above suggestion, @sgugger

**Comment by sgugger** [comment]
So please click the button to commit it.

**Comment by kant** [comment]
confirmed commit @sgugger

**Pull request merged**